src/HistoryEnter/page/index.tsx

```diff
@@ -8,28 +9,44 @@ import {
 import Header from '../../components/common/Header';
 import * as S from './HistoryEnter.style';
 
-const HistoryEnterList = [
-  {
-    title: '즐겨찾기한 레큐북',
-    variant: 'book',
-    image: <ImgMypageFavoriteLecueBook />,
-  },
-  {
-    title: '내가 만든 레큐북',
-    variant: 'book',
-    image: <ImgMypageMakeLecueBook />,
-  },
-  { title: '내가 남긴 레터', variant: 'letter', image: <ImgMypageLetter /> },
-];
-
 function HistoryEnter() {
+  const navigate = useNavigate();
+
+  const HistoryEnterList = useMemo(
+    () => [
+      {
+        title: '즐겨찾기한 레큐북',
+        variant: 'book',
+        image: <ImgMypageFavoriteLecueBook />,
+        handleClickTab: () => navigate('/mypage/history', { state: 1 }),
```

Comment:
P1: 아래처럼 수정하면 0도 잘 받아와지긴 하네요..!!
객체화하지 않고 숫자 0만 보냈을 때는 왜 잘 안되는지 .. . 이유는 저도 더 찾아보겠습니당 .. ㅜ_ㅜ

```suggestion
        handleClickTab: () => navigate('/mypage/history', { state: { option: 0 } }),
```